gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# Interested/Going on Profile and background on match.html and logout.h…

### Description

Now Interested/Going events are shown on User Profile and also fixed the grandma's picture bug so that heroku recognizes the pictures

### Testing Directions

```
python manage.py makemigrations
python manage.py migrate
python manage.py pull_seatgeek_data
python manage.py make_users
```

Mark a user interested/going to events, Add more people to your squad, See the profile, it now displays the events

### Checklist
